### PR TITLE
Initial changes for Optimize 8.3/3.11

### PR DESCRIPTION
**Pull request by akeller**
## Description

Some assistances with the changes mentioned here - https://github.com/camunda/product-hub/issues/1247#issuecomment-1735492171

## When should this change go live?

- [ ] This change is not yet live and should not be merged until {ADD_DATE} (apply `hold` label or convert to draft PR)?
- [x] There is no urgency with this change.
- [ ] This change or page is part of a marketing blog, conference talk, or something else on a schedule.
- [ ] This functionality is already available but undocumented.
- [ ] This is a bug fix or security concern.

## PR Checklist

- [ ] I have added changes to the relevant `/versioned_docs` directory, or they are not for an **already released version**.
- [ ] I have added changes to the main `/docs` directory (aka `/next/`), or they are not for **future versions**.
- [ ] My changes require an [Engineering review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [ ] My changes require a [technical writer review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

**Comment by akeller** [comment]
Even though the page URL changed, no redirects are needed since this is in `/next/`.

File: docs/reference/supported-environments.md [hunk 1]
```diff
@@ -78,17 +78,17 @@ You can also use newer versions of Desktop and Web Modeler with older Zeebe vers
...
+| Optimize 3.6.x          | Camunda Platform 7.14.x, 7.15.x, 7.16.x         | OpenJDK 11+               | 7.8.0+, 7.9.0+, 7.10.0+, 7.11.0+, 7.12.0+, 7.13.0+, 7.14.0+, 7.15.0+          |
+| Optimize 3.7.x          | Camunda Platform 7.14.x, 7.15.x, 7.16.x         | OpenJDK 11+               | 7.8.0+, 7.9.0+, 7.10.0+, 7.11.0+, 7.12.0+, 7.13.0+, 7.14.0+, 7.15.0+, 7.16.2+ |
+| Optimize 3.8.x          | Camunda Platform 7.15.x, 7.16.x, 7.17.x         | OpenJDK 11+               | 7.10.0+, 7.11.0+, 7.12.0+, 7.13.0+, 7.14.0+, 7.15.0+, 7.16.2+, 7.17.0+        |
+| Optimize 3.9.x          | Camunda Platform 7.16.x, 7.17.x, 7.18.x         | OpenJDK 11+               | 7.13.0+, 7.14.0+, 7.15.0+, 7.16.2+, 7.17.0+                                   |
+| Optimize 3.10.x         | Camunda Platform 7.17.x, 7.18.x, 7.19.x         | OpenJDK 17+               | 7.16.2+, 7.17.0+, 8.5.0+, 8.6.0+                                              |
+| Optimize 8.3.x / 3.11.x | Camunda Platform 7.17.x, 7.18.x, 7.19.x, 7.20.x | OpenJDK 17+               | 8.7.0+, 8.8.0+                                                                |
```
Comment (RomanJRW) [inline docs/reference/supported-environments.md:91]: Ooh I just noticed that the Engine support listed is wrong. Would you mind removing 7.17.x support for 3.11 please?

File: optimize/self-managed/optimize-deployment/migration-update/3.10-to-3.11_8.3.md [hunk 2]
```diff
@@ -1,13 +1,13 @@
 ---
-id: 3.10-to-7.20_8.3
-title: "Update notes (3.10 to 7.20/8.3)"
+id: 3.10-to-3.11_8.3
```
Comment (RomanJRW) [inline optimize/self-managed/optimize-deployment/migration-update/3.10-to-3.11_8.3.md:2]: I am pretty sure I added a redirect to this file in `.htaccess`. Does it need changing/removing?

**Review by RomanJRW — DISMISSED** [review]
Nice, thanks for the proactive help with this. Changes LGTM, just a couple of minor things ✅

File: optimize/self-managed/optimize-deployment/migration-update/3.10-to-3.11_8.3.md (hunk 2 above)
Comment (akeller) [reply optimize/self-managed/optimize-deployment/migration-update/3.10-to-3.11_8.3.md:2]: Oops! We shouldn't need it, but I'll have a look.

**Review by christinaausley — APPROVED** [review]

File: docs/reference/supported-environments.md [hunk 3]
```diff
@@ -78,17 +78,17 @@ You can also use newer versions of Desktop and Web Modeler with older Zeebe vers
...
+| Optimize 3.6.x          | Camunda Platform 7.14.x, 7.15.x, 7.16.x    | OpenJDK 11+               | 7.8.0+, 7.9.0+, 7.10.0+, 7.11.0+, 7.12.0+, 7.13.0+, 7.14.0+, 7.15.0+          |
+| Optimize 3.7.x          | Camunda Platform 7.14.x, 7.15.x, 7.16.x    | OpenJDK 11+               | 7.8.0+, 7.9.0+, 7.10.0+, 7.11.0+, 7.12.0+, 7.13.0+, 7.14.0+, 7.15.0+, 7.16.2+ |
+| Optimize 3.8.x          | Camunda Platform 7.15.x, 7.16.x, 7.17.x    | OpenJDK 11+               | 7.10.0+, 7.11.0+, 7.12.0+, 7.13.0+, 7.14.0+, 7.15.0+, 7.16.2+, 7.17.0+        |
+| Optimize 3.9.x          | Camunda Platform 7.16.x, 7.17.x, 7.18.x    | OpenJDK 11+               | 7.13.0+, 7.14.0+, 7.15.0+, 7.16.2+, 7.17.0+                                   |
+| Optimize 3.10.x         | Camunda Platform 7.17.x, 7.18.x, 7.19.x    | OpenJDK 17+               | 7.16.2+, 7.17.0+, 8.5.0+, 8.6.0+                                              |
+| Optimize 8.3.x / 3.11.x | Camunda Platform 7.18.x, 7.19.x, 7.20.x    | OpenJDK 17+               | 8.7.0+, 8.8.0+                                                                |
```
[the remark this answers is not in the document]
Comment (christinaausley) [reply docs/reference/supported-environments.md:91]: @pepopowitz Do you have strong opinions here?

[the remark this answers is not in the document]
Comment (pepopowitz) [reply docs/reference/supported-environments.md:91]: What @RomanJRW says makes sense to me. It's clearly a table for C7 users, and the 8.3 would probably create more confusion.

[the remark this answers is not in the document]
Comment (OmranAbazid) [reply docs/reference/supported-environments.md:91]: I will push a change to adjust this. 
@christinaausley Can you take a review the changes that i added.

**Review by christinaausley — APPROVED** [review]

**Review by christinaausley — DISMISSED** [review]

**Review by OmranAbazid — APPROVED** [review]

**Pull request merged**